db/seed.sql

```diff
@@ -0,0 +1,14 @@
+TRUNCATE TABLE paintings CASCADE;
+TRUNCATE TABLE artists CASCADE;
+
```

Comment:
This is great, but I think the assignment was asking for seeds to be done in Ruby. Either way, great practice!